provider_test.go

```diff
@@ -253,45 +253,45 @@ func ExampleParseProviderSource() {
 		log.Fatal(err)
 	}
 	fmt.Printf("%#v", pAddr)
-	// Output: tfaddr.Provider{Type:"aws", Namespace:"hashicorp", Hostname:svchost.Hostname("registry.terraform.io")}
+	// Output: tfaddr.Provider{Type:"aws", Namespace:"hashicorp", Hostname:svchost.Hostname("registry.opentf.org")}
 }
 
 func TestParseProviderSource(t *testing.T) {
 	tests := map[string]struct {
 		Want Provider
 		Err  bool
 	}{
-		"registry.terraform.io/hashicorp/aws": {
+		"registry.opentf.org/hashicorp/aws": {
 			Provider{
 				Type:      "aws",
 				Namespace: "hashicorp",
 				Hostname:  DefaultProviderRegistryHost,
 			},
 			false,
 		},
-		"registry.Terraform.io/HashiCorp/AWS": {
+		"registry.Opentf.org/HashiCorp/AWS": {
 			Provider{
 				Type:      "aws",
 				Namespace: "hashicorp",
 				Hostname:  DefaultProviderRegistryHost,
 			},
 			false,
 		},
-		"terraform.io/builtin/terraform": {
+		"opentf.org/builtin/opentf": {
 			Provider{
-				Type:      "terraform",
+				Type:      "opentf",
 				Namespace: BuiltInProviderNamespace,
 				Hostname:  BuiltInProviderHost,
 			},
 			false,
 		},
 		// v0.12 representation
-		// In most cases this would *likely* be the same 'terraform' provider
+		// In most cases this would *likely* be the same provider
 		// we otherwise represent as builtin, but we cannot be sure
 		// in the context of the source string alone.
-		"terraform": {
+		"opentf": {
```

Comment:
This shouldn't be changed I think, the terraform default built-in provider stays I believe.

Reply:
Rolled back